content/guidelines/reviewing.en.md

```diff
@@ -0,0 +1,109 @@
+
+### Step-by-step Review Guide
+
+1. Click on the inspect the declaration suggestion link to view contribution using the contribution tool.
+2. Use the link provided in the URL section of the contribution tool to check out the original document.
+3. Verify that the name of the service matches the JSON file and complies with the [guidelines](https://docs.opentermsarchive.org/guidelines/declaring/#service-name).
+4. Quickly scan the document to ensure that the correct term type has been selected. To determine the term type, consider who the intended audience is and what the document is discussing. You can also refer to the [terms type guide](https://github.com/OpenTermsArchive/terms-types/blob/main/termsTypes.json) to find the best term type for the document.
+5. Confirm that the selected area of the document contains only one term type and does not include any other types.
+6. Check both the significant and insignificant parts of the document.
+    - Ensure that only [simple/basic selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#basic_selectors) are used and avoid complex selectors like [combinators](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#combinators) and [pseudo selectors](https://www.w3schools.com/css/css_pseudo_elements.asp). To simplify, try to avoid selectors with indexes/numbers e.g `:nth-child(n)` and long chains e.g. `div > body > .aside > p > span > a )`
+    - Ensure that the significant parts do not include a table of contents, contact links, or other insignificant details that may cause confusion by giving a false impression of change.
+7. Verify the version of the document in the contribution tool by clicking on the `verify version` button.
+8. Ensure that all checks generated by the OTA-bot are manually checked.
+9. When you confirm that the term contribution has been made to the correct collection, proceed to add a review.
+10. Merge the contribution.
+
+## How To Review Pull Requests that Update Declarations
+
+When a declaration can no longer be tracked by the Open Terms Archive engine, an issue is created in the collection. This issue contins the details on why the declaration cannot be tracked and the date the challenge was encountered.  
+Contributors are then required to help make this fixes to the declaration in order to maintain the declaration's tracking.
+The updates can be made using the contribution tool and it's effects will be similar to the one seen when adding declaration but with a slight change.
+The pull request created will consist of fewer checks than those that add declarations. The checks will guide the reviewer on the key things to look out for during the review process, and a link to inspect the declaration.
+
+For PRs that update declarations, you should focus should be on two things: history file and the declaration.
```

Comment:
We have not used the “PR” short term until now, let's avoid using it altogether if we can, in order to limit the complexity 🙂 
```suggestion
For pull requests that update declarations, you should focus should be on two things: history file and declaration.
```